app/assets/stylesheets/moderators.scss

```diff
@@ -379,7 +384,7 @@
       color: var(--accent-brand);
 
       &:after {
-        content: 'Marked as abuse/spam';
+        content: 'Flagged to Admins';
```

Comment:
Maybe "Flagged for Admin Review" that might be too long. Just throwing out some ideas. Flagged "to" Admin feels a little weird but it's not bad.